docs/src/examples/components/List/Types/ListExampleNavigbale.shorthand.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
```

Comment:
This file has misspelled name -> does not show up in docsite docs/src/examples/components/List/Types/ListExampleNavig**ba**le.shorthand.tsx

Reply:
good catch, thanks!